### ISSUE TITLE
TUTORIAL: Other Open/Source Free Tools

### ISSUE THREAD

**Issue by jongund** [teachaccess/tutorial#3]
Could the following open source (free) tools also be included int he list of tools:

1. Functional Accessibility Evaluator 2.0
https://fae.disability.illinois.edu 

2. AInspector Sidebar for Firefox
http://ainspector.github.io 

Both of these tools are based on the Open Source OpenAjax evaluation library that evaluates web pages for WCAG 2.0 Level A and AA requirements.

**Comment by TheJaredWilcurt** [comment]
Pa11y is a good tool for developers to automate checking for Section 508 and WCAG 2.0 A/AA/AAA violations.

* http://pa11y.org

Koa11y is built on top of Pa11y to be an easy tool for non-devs to test pages.

* https://open-indy.github.io/Koa11y
